codeship-steps.yml

```diff
@@ -6,5 +6,11 @@
   steps:
     #-  command: ./.ci/travis/travis.sh site
     #-  command: ./.ci/travis/travis.sh nondex
-    -  command: ls -la
-    -  command: .ci/run-link-check-plugin.sh
+    - command: |
+        ls -la
+        SKIP_FILES1=".github|distelli-manifest.yml|fast-forward-merge.sh"
+        SKIP_FILES2="LICENSE|LICENSE.apache20|README.md|release.sh|RIGHTS.antlr"
+        SKIP_FILES3="shippable.yml|wercker.yml|appveyor.yml|circle.yml"
+        SKIP_FILES="$SKIP_FILES1$SKIP_FILES2$SKIP_FILES3"
+        if [[ $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c) > 0 ]];
+        then .ci/run-link-check-plugin.sh; fi
```

Comment:
Please move `then` and `fi` to separate lines, we respect indentation rules even in bash

Reply:
Done 👍